### PR TITLE
feat(builder): dynamic questions based on actionType

### PR DESCRIPTION
**Pull request by mmackz**
This updates the plugin builder to ask dynamic questions based on the action type and builds the test-transactions.ts file based on input given by the user.

It uses viem to grab the details of the transaction and the tokendetails (symbol/decimals) and uses this info to build test cases.

After this update, the user should only need to create the transaction filter and generate a changeset.

includes #220

### CODE REVIEW

**Comment by Quazia** [comment]
@mmackz Is this good for review? If so I'll try and get it tackled tomorrow depending how I'm doing with the dynamic names work I'm on right now.

**Comment by mmackz** [comment]
> @mmackz Is this good for review? If so I'll try and get it tackled tomorrow depending how I'm doing with the dynamic names work I'm on right now.

Yes please 🙏

**Comment by mmackz** [comment]
Builder Bugs 

- Chains should only be imported if at least one chain is selected (Project.ts)
-   GreaterThanOrEqual should only be imported if there is a valid amount. (test-transactions.ts)
- Some issues with GreaterThan or Equal on vote action and others with amounts that are not swap or bridge.

**Review by gnomadic — APPROVED** [review]

File: apps/create-plugin/template/src/test-transactions.ts.t [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  Chains{{/eq}}
+} from '@rabbitholegg/questdk-plugin-utils'
+{{else}}
+import { type TestParams } from '@rabbitholegg/questdk-plugin-utils'
+{{/if}}
+
+{{#unless tx.length}}
+export const {{uppercase actionType}}_TEST: TestParams<{{capitalize actionType}}ActionParams> = {
+  transaction: {
+    chainId: '',
```
Comment (gnomadic) [inline apps/create-plugin/template/src/test-transactions.ts.t:23]: can we set these with valid typed default values so it compiles if the user doesn't provide a tx?

File: packages/apples/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Apples Plugin for Boost
+
+### New Plugin TODO list
```
Comment (gnomadic) [inline packages/apples/README.md:3]: whoops also delete this test output :)

File: apps/create-plugin/template/src/test-transactions.ts.t (hunk 1 above)
Comment (mmackz) [reply apps/create-plugin/template/src/test-transactions.ts.t]: FIxed!

**Pull request merged**